config/app_dynamics_agent.yml

```diff
@@ -17,6 +17,6 @@
 ---
 version: 4.1.+
 repository_root: ! '{default.repository.root}/app-dynamics'
-default_application_name:
-default_node_name: ! '$(expr "$VCAP_APPLICATION" : ''.*instance_index[": ]*\([[:digit:]]*\).*'')'
-default_tier_name:
+default_application_name: 
```

Comment:
The empty spaces here and line 22 shouldn't be added.  You're aiming with a diff that only has a single line changed.